helper/iofs/iofs.go

```diff
@@ -0,0 +1,142 @@
+// Package iofs provides an adapter from billy.Filesystem to a the
+// standard library io.fs.FS interface.
+package iofs
+
+import (
+	"io"
+	"io/fs"
+	"path/filepath"
+
+	billyfs "github.com/go-git/go-billy/v5"
+	"github.com/go-git/go-billy/v5/helper/polyfill"
+)
+
+// Wrap adapts a billy.Filesystem to a io.fs.FS.
+func Wrap(fs billyfs.Basic) fs.FS {
+	return &adapterFs{fs: polyfill.New(fs)}
+}
+
+type adapterFs struct {
+	fs billyfs.Filesystem
+}
+
+var _ fs.FS = (*adapterFs)(nil)
+var _ fs.ReadDirFS = (*adapterFs)(nil)
+var _ fs.StatFS = (*adapterFs)(nil)
+var _ fs.ReadFileFS = (*adapterFs)(nil)
+
+// GlobFS would be harder, we don't implement for now.
```

Comment:
```suggestion
// TODO: Add GlobFS support.
```

Reply:
Reformatted to a TODO, though my $0.02 would be to not implement GlobFS until someone discovers they actually need it.